### PR TITLE
explicitly set readthedocs template

### PR DESCRIPTION
**Pull request by nikita-fuchs**
This somehow got lost during @davidyuk 's last rebase and therefore the readTheDocs looked extremely ugly, using mkdocs default theme. But locally it's somehow loading the sphinx theme automagically - how is that actually done @davidyuk  ?
**Edit**: No, it's not automagically set for me anymore, maybe it was just a caching issue.

As a quickfix, I've redirected readTheDocs to the feature branch for now and will reconfigure to `devlop` after merge. thanks for a quick review ! ;)

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -6,6 +6,15 @@ plugins:
...
 repo_url: 'https://github.com/aeternity/aepp-sdk-js'
 
+theme:
+  name: readthedocs
+  highlightjs: true
+  hljs_languages:
+    - yaml
+    - erlang
+    - javascript
+    - js
```
Comment (davidyuk) [inline mkdocs.yml:16]: I have made some tests before and I didn't find any difference between this and default settings.
```suggestion
```

File: mkdocs.yml [hunk 2]
```diff
@@ -6,6 +6,15 @@ plugins:
         on_pre_build: "docs.hooks:pre_build"
 repo_url: 'https://github.com/aeternity/aepp-sdk-js'
 
+theme:
+  name: readthedocs
```
Comment (davidyuk) [inline mkdocs.yml:10]: didn't you notice that the navigation is broken when using this theme? Or it is already fixed somehow? https://github.com/aeternity/aepp-sdk-js/pull/1077#discussion_r514587519

File: mkdocs.yml (hunk 1 above)
Comment (nikita-fuchs) [reply mkdocs.yml:16]: okay

File: mkdocs.yml (hunk 2 above)
Comment (nikita-fuchs) [reply mkdocs.yml:10]: I'm writing an issue on this right now, thing is that the default theme is very ugly and unusable, and people participating in a currently ongoing hackathon mind that more than the buggy nav 😄 I'm proposing solutions, hang in there.

Comment (davidyuk) [reply mkdocs.yml:10]: I think the broken navigation worse than a theme that doesn't look enough nice 🤷‍♀️

**Comment by omar-saadoun** [comment]
Hi @davidyuk it would of great help getting this merged. Documentation plays an important role and there is a lot of work being done. Still a challenge as we have yet lot of missing elements on documentation but we are on the right track.

**Comment by davidyuk** [comment]
@mradkov I don't have anything to add, feel free to approve this if it looks nice to you.

**Comment by nikita-fuchs** [comment]
Hold on, this is not the latest state of things anymore. Wait for tomorrow
pleas :)

Denis Davidyuk ***@***.***> schrieb am So., 28. März 2021,
20:22:

**Comment by mradkov** [comment]
@nikita-fuchs whats the state of this?

**Comment by nikita-fuchs** [comment]
@mradkov I've prepared a "new way of building the docs" in the branch readTheDocs-iris branch, which bundlesthe previous work as well as updated documentation. I will create a new PR beginning of next week.

**Pull request closed without merging**